activerecord/lib/arel/insert_manager.rb

```diff
@@ -38,6 +38,11 @@ def insert(fields)
       self
     end
 
+    def comment(*values)
```

Comment:
Is this internal API necessary (no way to use it by public API (Active Record))?